core/chains/evm/config/toml/config.go

```diff
@@ -521,6 +521,7 @@ func (a *Automation) setFrom(f *Automation) {
 type Workflow struct {
 	FromAddress      *types.EIP55Address `toml:",omitempty"`
 	ForwarderAddress *types.EIP55Address `toml:",omitempty"`
+	GasLimit         uint64
```

Comment:
nit: Should this be called DefaultGasLimit to indicate we can still overwrite in the workflow spec?

Reply:
sorted